iroha/src/tx.rs

```diff
@@ -84,6 +84,11 @@ impl Transaction {
             signatures,
         } = self
         {
+            for signature in &signatures {
+                if signature.verify(&Vec::from(&request)).is_err() {
```

Comment:
Message that we verify should be scale encoded transaction. Is it the case here?

Reply:
@kamilsa yes - we do not expose `scale` to have an ability to replace with any other binary format. right now we use `Into\From for Vec<u8>` and inside this impls we use `scale`